daemon/daemon.go

```diff
@@ -779,7 +779,7 @@ func NewDaemon(dp datapath.Datapath) (*Daemon, *endpointRestoreState, error) {
 	debug.RegisterStatusObject("ipam", d.ipam)
 
 	bootstrapStats.k8sInit.Start()
-	k8s.Configure(option.Config.K8sAPIServer, option.Config.K8sKubeConfigPath)
+	k8s.Configure(option.Config.K8sAPIServer, option.Config.K8sKubeConfigPath, 0, 0)
```

Comment:
Why not `k8s.GetQPS()` and `k8s.GetBurst()` here?

Reply:
Setting to the defaults instead since `Configure` is the thing that sets the values of `k8s.GetQPS` and `k8s.GetBurst()`